tests/system/python/scripts/install_python_plugin_remote

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+set -e
+
+__author__="Vaibhav Singhal"
+__copyright__="Copyright (c) 2019 Dianomic Systems"
+__license__="Apache 2.0"
+__version__="1.0.0"
+
+###########################################################################################################
+# Usage text for this script
```

Comment:
can not we just copy the install_python_plugin script to remote and execute that; Also the exitCode of that should give us info for successful execution.

Reply:
Done. Nice suggestion!